client/blocks/product-selector/index.jsx

```diff
@@ -143,10 +144,21 @@ export class ProductSelector extends Component {
 		} );
 	}
 
-	handleCheckoutForProduct = productObject => {
-		const { selectedSiteSlug } = this.props;
+	handleCheckoutForProduct = ( productObject, product ) => {
```

Comment:
I must admit that this change makes this method a bit confusing. Would be nice to be able to do this without adding this additional argument. Is there other way to get the current product purchase slug?